lib/modules/manager/flux/extract.ts

```diff
@@ -197,6 +207,29 @@ function resolveResourceManifest(
         deps.push(dep);
         break;
       }
+      case 'OCIRepository': {
+        const container = resource.spec.url?.replace('oci://', '');
+        const dep: PackageDependency<FluxManagerData> = {
+          depName: container,
+        };
+        if (resource.spec.ref?.digest) {
+          dep.currentDigest = resource.spec.ref.digest;
```

Comment:
i would still log a debug message when tag is also found.
should be something like `digest and tag found, ignoring tag`

Reply:
Fixed in upcoming commit